### DISCUSSION TITLE
Actually we can use voice without HTTPS on chrome and brave

### DISCUSSION THREAD

**Discussion by thiswillbeyourgithub** [open-webui/open-webui#4352] [Show and tell]
Hello,

I've read everywhere in the documentation, discussions, and other GitHub issues that it was impossible to make Voice Recognition, STT, or Speech Recognition work without enabling HTTPS, because modern browsers prevent it. However, Chrome, Brave, Chromium, etc. all allow adding a specific domain to the exceptions, even though this feature is quite hidden and it's widely reported as impossible. It even works on mobile.

To do this, follow the instructions at this address: https://medium.com/@Carmichaelize/enabling-the-microphone-camera-in-chrome-for-local-unsecure-origins-9c90c3149339

Basically:
1. Open [chrome://flags/#unsafely-treat-insecure-origin-as-secure](chrome://flags/#unsafely-treat-insecure-origin-as-secure)
2. Add your local ip there: `192.168.1.39:3000` for example
3. Restart the browser

Of course it's a security tradeoff, but in my case I'm using a VPN to reach this server so it makes sense.

Ping #1148 #3765 #4020 

I think this should be added in the FAQ, let the user decide the tradeoff it wants!

Have a great day everyone, goodbye.

**Comment by justinh-rahb** [comment]
PR welcome to the Docs repo: https://github.com/open-webui/docs

**Reply by thiswillbeyourgithub** [reply]
Thanks, I'll do it in the next few days.

**Reply by thiswillbeyourgithub** [reply]
Done at https://github.com/open-webui/docs/pull/172
Thanks!

**Comment by Haervwe** [comment]
Worked like a charm, thanks a lot! Took me a lot of googling to get here
